### DISCUSSION TITLE
合集框的夜间模式没有匹配

### DISCUSSION THREAD

**Discussion by IOAOl** [the1812/Bilibili-Evolved#3090] [提问 / Q&A] [answered]
显示如下
![image_1](https://user-images.githubusercontent.com/49601034/156889672-60c708d8-c88a-4b4b-910f-6a40c0c83b9b.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
圆角看着像是新版，新版播放页的话是还没适配的：#2615
旧版页面的话重复：<https://github.com/the1812/Bilibili-Evolved/issues/2136#issuecomment-1021020336>，而且大体上是已适配的

**Reply by IOAOl** [reply]
新版播放页的，近期有适配的打算吗

**Reply by the1812** [reply]
没有，建议返回旧版
